parameter_float: 0.75
parameter_bool: false
parameter_string: "Hello World!"
